karrot/activities/tests/test_tasks.py

```diff
@@ -54,7 +54,7 @@ def test_activity_reminder_notifies_subscribers(self, notify_subscribers_by_devi
             kwargs['click_action'],
         )
         self.assertIn(
-            'Upcoming pickup',
+            'Upcoming {}'.format(self.activity.typus.name),
```

Comment:
Some languages might have different noun forms so this kind of translation is too limited.